dist/pouchdb.quick-search.js

```diff
@@ -275,6 +275,7 @@ exports.search = utils.toPromise(function (opts, callback) {
         docIdsToFieldsToQueryTerms, docIdsToFieldsToNorms, fieldBoosts);
       return rows;
     }).then(function (rows) {
+      total_rows = rows.length;
```

Comment:
This doesn't seem to implement `total_rows` correctly. `total_rows` is not the total number of documents returned; that would be redundant because you can just call `docs.length`. :) `total_rows` is the total number of *possible* documents that *could* be returned, e.g. if you are doing pagination. In fact the whole point of `total_rows` is to make it easier to do pagination, so you know how many documents remain to be read.